source/imt-terms.ttl

```diff
@@ -52,11 +57,26 @@
     skos:inScheme </term/imt> ;
     skos:exactMatch iana:text\/turtle .
 
+<text/csv> a :EncodingFormat ;
+    skos:notation "text/csv" ;
+    skos:inScheme </term/imt> ;
+    skos:exactMatch iana:text\/csv .
+
+<text/tab-separated-values> a :EncodingFormat ;
+    skos:notation "text/tab-separated-values" ;
+    skos:inScheme </term/imt> ;
+    skos:exactMatch iana:text\/tab-separated-values .
+
 <text/xml> a :EncodingFormat ;
     skos:notation "text/xml" ;
     skos:inScheme </term/imt> ;
     skos:exactMatch iana:text\/xml .
 
+<video/JPEG> a :EncodingFormat ;
```

Comment:
This should probably be `image/jpeg` instead, but I can´t find it among IANA´s media types: https://www.iana.org/assignments/media-types/media-types.xhtml 🤷

Reply:
Jag, hittar inte heller. Men i I tabellen av imagetyper länkar iana till [denna](https://datatracker.ietf.org/doc/html/rfc2046) under 'jpeg':, och där i nämns jpeg flera gånger och [här](https://datatracker.ietf.org/doc/html/rfc2046#section-5.2.3) så ger de även ett exempel med just `/image/jpeg` så det verkar som att man köra på det.